### PR TITLE
relicense to Apache License v2

### PR DESCRIPTION
**Pull request by szha**
Signed-off-by: Sheng Zha <zhasheng@amazon.com>

### CODE REVIEW

**Comment by prasanthpul** [comment]
This is part of the ONNX-wide relicensing that has already been done by the other converter repos.

File: skl2onnx/algebra/automation.py [hunk 1]
```diff
@@ -1,8 +1,5 @@
-# -------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-# --------------------------------------------------------------------------
+# SPDX-License-Identifier: Apache-2.0
```
Comment (prasanthpul) [inline skl2onnx/algebra/automation.py:1]: @szha I noticed the copyrights in many files were removed as part of the license change. Was it on purpose or unintentionial?

Comment (szha) [reply skl2onnx/algebra/automation.py:1]: This is intentional. We can include the copyright notices into a unified NOTICE file for simplicity.

**Review by xadupre — APPROVED** [review]

**Comment by faxu** [comment]
@szha @xadupre  this also needs to be updated in the documentation on this page: http://onnx.ai/sklearn-onnx/

**Comment by szha** [comment]
@faxu where is the source of this website?

**Comment by faxu** [comment]
I believe it's here: https://github.com/onnx/sklearn-onnx/blob/gh-pages/index.html

But it should be generated using the .rst file [here](https://github.com/onnx/sklearn-onnx/blob/master/docs/index.rst) which already reflects the license update. 

@xadupre can you help to regenerate the html pages based on the latest updates?

**Comment by xadupre** [comment]
Source of the website are in branch gh-pages. Documentation is generated using sphinx (same as onnxruntime). You can use the command line ``python -m sphinx -j1 -v -N -T -b html -d build/doctrees docs dist/html``.

**Pull request merged**